app/views/govuk_publishing_components/components/_highlight_boxes.html.erb

```diff
@@ -1,11 +1,14 @@
 <%
   items ||= []
   inverse ||= false
+  within_multitype_list ||= false
   inverse_class = "gem-c-highlight-boxes--inverse" if inverse
   highlight_boxes_helper = GovukPublishingComponents::Presenters::HighlightBoxesHelper.new(local_assigns)
 %>
 <% if items.any? %>
-  <ol class="gem-c-highlight-boxes" <%= "data-module=track-click" if highlight_boxes_helper.data_tracking? %>>
+  <% unless within_multitype_list %>
+    <ol class="gem-c-highlight-boxes" <%= "data-module=track-click" if highlight_boxes_helper.data_tracking? %>>
```

Comment:
This isn't really part of this PR, but is there a way of writing this line that github will like?

Reply:
I haven't been able to find much info on this - I tried a few things but nothing seemed to fix it.

Reply:
Fair enough. I'll take a look at the rest shortly.